orc8r/cloud/go/obsidian/access/rest_middleware.go

```diff
@@ -24,14 +24,29 @@ import (
 	"magma/orc8r/cloud/go/obsidian"
 	"magma/orc8r/cloud/go/services/accessd"
 	accessprotos "magma/orc8r/cloud/go/services/accessd/protos"
+	"magma/orc8r/cloud/go/services/certifier"
+	certprotos "magma/orc8r/cloud/go/services/certifier/protos"
 	merrors "magma/orc8r/lib/go/errors"
 )
 
-// Access Middleware:
+// NoBasicAuthEndpoints include all REST API endpoints that do not require
+// the Basic Auth header to access.
+type NoBasicAuthEndpoints struct {
+	endpoints map[string]bool
+}
+
+func InitNoBasicAuthEndpoints() NoBasicAuthEndpoints {
+	return NoBasicAuthEndpoints{endpoints: map[string]bool{
+		// Users should be able to access their own authentication information
+		"/magma/v1/user/login": true,
+	}}
+}
+
```

Comment:
This is weird lol. Just use a bare map

```go
unprotectedPaths := map[string]interface{}{}{
	...
}
```

Reply:
Ok aha I was confused to how to make a package-level variable, apparently have to put `var` in front of it.